src/locales/de/app.json

```
@@ -0,0 +1,132 @@
+{
+	"loading": "Lade...",
+	"download-ready": "Jetzt herunterladen!",
+	"navigation": {
+		"home": "Start",
+		"configuration": "Kalender erstellen",
+		"features": "Features",
+		"faq": "FAQ"
+	},
+	"language": {
+		"label": "Sprache",
+		"english": "Englisch",
+		"polish": "Polnisch",
+		"french": "Französisch",
+		"spanish": "Spanisch",
+		"german": "Deutsch"
+	},
+	"configuration": {
+		"selector": {
+			"label": "Start",
+			"template": {
+				"label": "Wähle eine Vorlage aus:",
+				"basic": "Einfach",
+				"advanced": "Fortschritten",
+				"blank": "Leer",
+				"minimalistic": "Minimalistisch"
+			},
+			"upload": {
+				"label": "Lade einen bestehendes ReCalendar PDF hoch um die Konfiguration zu übernehmen:",
+				"loading": "Lese Konfiguration aus dem PDF, bitte warten...",
+				"error": "Es konnte keine Konfiguration aus dem PDF übernommen werden - bist du sicher, dass das PDF mit dieser Website generiert worden ist?",
+				"success": "Konfiguration aus bestehendem PDF geladen - Willkommen zurück!"
+			}
+		},
+		"general": {
+			"label": "Allgemeine Optionen",
+			"year": "Jahr",
+			"starting-month": {
+				"label": "Start-Monat",
+				"description": "Das erste Monat, welches im Kalender enthalten seinsoll. Du kannst zB Oktober auswählen um einen Kalender für ein Semster, statt für ein Jahr zu erstellen."
+			},
+			"month-count": {
+				"label": "Anzahl der Monate",
+				"description": "Für wie viele Monate möchtest du einen Kalender erstellen?"
+			},
+			"first-day-of-week": "Erster Tag der Woche",
+			"weekend": "Wochenende"
+		},
+		"button": {
+			"refresh": "Vorschau aktualisieren",
+			"download": "Kalender erstellen und herunterladen",
+			"generating": "Kalender wird erstellt, bitte warten..."
+		},
+		"itinerary": {
+			"empty": "Leerer Zeitplan",
+			"placeholder": {
+				"lines": "Zeilenanzahl",
+				"page": "Neue Seite"
+			},
+			"button": {
+				"copy": "Zeitplan für die komplette Woche übernehmen",
+				"remove": "Entfernen",
+				"item": "Eintrag hinzufügen",
+				"page": "Neue Seite hinzufügen",
+				"lines": "Leere Zeilen hinzufügen"
+			}
+		},
+		"items-list": {
+			"button": {
+				"item": "Eintrag hinzufügen",
+				"remove": "Entfernen"
+			},
+			"empty": "Leere Liste",
+			"placeholder": "Eintrag"
+		},
+		"toggle-form": {
+			"enabled": "Aktiviert",
+			"disabled": "Deaktiviert"
+		},
+		"month": {
+			"title": "Monatsübersicht",
+			"description": "In der Monatsübersicht kannst du Ziele für das kommende Monat festhalten, deine Gewohnehiten tracken oder andere Pläne aufzeichnen.",
+			"habits": {
+				"title": "Gewohnheiten"
+			},
+			"itinerary": {
+				"title": "Zeitplan"
+			}
+		},
+		"week": {
+			"title": "Wochenübersicht",
+			"description": "Die Wochenübersicht gibt dir Vogelperspektive über deine Woche und macht die Planung einfacher.",
```

Übersicht and Überblick are actually more like synonyms so one could question the point of thelong sentence, but the expression of Vogelperspektive does not make much sense in German here. I am no native English speaker, but even the English translation reads a little unconventional to me 

```suggestion
			"description": "Die Wochenübersicht gibt dir einen Überblick über deine Woche und macht die Planung einfacher.",
```